config/betterquesting/DefaultQuests/Quests/KillAlltheThings-AAAAAAAAAAAAAAAAAAAAJQ==/KillEnderDragon-AAAAAAAAAAAAAAAAAAABkA==.json

```diff
@@ -31,7 +31,7 @@
       "lockedProgress:1": 0,
       "autoClaim:1": 0,
       "isSilent:1": 0,
-      "desc:8": "The dragon must die. Long enough has he destroyed those precious endstone blocks!\n\n§3PS: Hold shift and whack it with a vanilla sword."
+      "desc:8": "The dragon must die. Long enough has she destroyed those precious endstone blocks!\n\n§3PS: Hold shift and whack it with a vanilla sword."
```

Comment:
I agree with the gender correction given the lore, but I think it is worth us reviewing this in general as the ender dragon cannot break endstone.

Reply:
Maybe something more along the lines of hoarding/protecting them or something else?

Reply:
HEE dragon breaks endstone, quest is correct